openreview/conference/templates/desk_reject_process.py

```diff
@@ -16,6 +16,9 @@ def process(client, note, invitation):
     if PAPER_AREA_CHAIRS_ID:
         committee.append(PAPER_AREA_CHAIRS_ID)
     committee.append(PROGRAM_CHAIRS_ID)
+    committee.append(CONFERENCE_ID)
```

Comment:
if program chairs are already in the committee you don't need to add the conference id. 

I think the ECCV invitation doesn't have this version.

Reply:
Right, the ECCV version is missing the reviewers and ACs. When this PR is merged, I can update the Desk_Reject and Withdraw invitations with this version.

Reply:
I don't think we need to merge this PR, we need to use the version of master where AC/Reviewers are readers of the withdrawn/desk rejected notes and they also get email notifications. 

We already hace assigned ACs and we are going to have assigned Reviewers. They should get email notifications.

Reply:
Agreed. I will close this PR since this change is not needed anymore.